### PR TITLE
New article firewall manager v2 port groups

### PR DESCRIPTION
**Pull request by stephamon**
Add editor feedback and images for #1925

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fc186c4be1/how-to/)

File: content/dedicated-hosting/firewall-manager-v2-port-groups.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+**Note:** The Firewall Manager v2 allows you to create port-groups using only TCP or UDP ports.
+
+### Why should I use a port-group?
+
+Port-groups improve the organization and readability of a firewall's running configuration. A running configuration that is easy to read and modify reduces the chances for a misconfiguration and increases the ability to troubleshoot issues quickly.
+
+### Port-groups in the Firewall Manager v2
+
+In Firewall Manager v2, you ca now view, modify, and delete any port-group on your firewall. In the previous version of the Firewall Manager, you were restricted to only modifying port-groups that began with the string *FWCP-*.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2-port-groups.md]: ... you can now view ...

File: content/dedicated-hosting/firewall-manager-v2-port-groups.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+1. Log in to the Firewall Manager v2 by following the steps in the [Firewall Manager v2](https://support.rackspace.com/how-to/firewall-manager-v2) article.
+
+2. In the navigation pane on the left side of the panel, click the firewall for which you want to see port-groups.
+
+3. Under **Rules** in the navigation pane, click **Port-groups**.
+
+   <img src="{% asset_path dedicated-hosting/firewall-manager-v2-port-groups/FWCPv2-port-group.png %}" alt="port-groups are located under rules in the left-hand navigation menu" />
+
+4. Scroll through or search the port-group list and click the appropriate port-group. The contents of the port-group are automatically displayed.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2-port-groups.md]: Scroll through (or search) the port-group list and click ...  [just a recommendation for clarity - not a requirement]

File: content/dedicated-hosting/firewall-manager-v2-port-groups.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+   Firewall Manager v2 organizes the port-groups in case-sensitive, alphabetical order. Therefore, port-group named with uppercase letters are displayed before those with lowercase letters. If you can't locate a port-group, search for the port-group by using your browser's search function.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2-port-groups.md]: ... Therefore, port-groups named ...

File: content/dedicated-hosting/firewall-manager-v2-port-groups.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+7. Click **Save Changes**.
+
+   Firewall Manager v2 interacts with your firewall and adds the configurations. This typically takes about one minute, depending on the size of your firewall's configuration.
+
+<img src="{% asset_path dedicated-hosting/firewall-manager-v2-port-groups/FWCPv2-add-port-group.png %}" alt="" />
+
+### Modify a port-group
+
+It is highly important that you understand the impact of modifying an existing port-group. Incorrectly modifying an existing port-group that is already referenced in an access-list has the potential to either create inappropriate network access or remove critical access.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2-port-groups.md]: ... or to remove critical ...

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-30ece8aa70/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fbf8935f50/how-to/)

**Pull request merged**